packages/location-field/src/styled.tsx

```diff
@@ -140,76 +141,61 @@ export const MenuItemHeader = styled.li`
   text-align: ${props => (props.centeredText ? "center" : "left")};
   white-space: nowrap;
 `;
-export const MenuItemLi = styled.li`
+export const MenuItemLi = styled.li<{ disabled?: boolean }>`
   &:hover {
     cursor: pointer;
     /* TODO: adjust highlight color based on props.color? */
     background-color: ${props => !props.disabled && "#f5f5f5"};
   }
 `;
 
-export class MenuItem extends Component {
-  onClick = () => {
-    const { disabled, onClick } = this.props;
+export const MenuItem = ({
+  active = false,
+  centeredText = false,
+  children,
+  // foregroundColor and backgroundColor would be preferred, but React has issues with
+  // these since they are style keywords
+  fgColor = null,
+  bgColor = null,
```

Comment:
Sort props.

Reply:
### Addressed in 18d5606109e7dfce7d00d42b42c8b1b6f075a0bc